docs/getting-started.md

```diff
@@ -0,0 +1,57 @@
+# Getting Started
+
+Before using `shiori`, make sure it has been installed on your system. By default, `shiori` will store its data in directory `$HOME/.local/share/shiori`. If you want to set the data directory to another location, you can set the environment variable `SHIORI_DIR` (`ENV_SHIORI_DIR` when you are before `1.5.0`) to your desired path.
+
+<!-- TOC -->
+
+- [Running Docker Container](#running-docker-container)
+- [How to use Shiori](#how-to-use-shiori)
+- [Deleting/Stopping Shiori](#deletingstopping-shiori)
+
+<!-- /TOC -->
+
+## Running Docker Container
```

Comment:
I have been giving a thought and I believe is better to point the installation to the proper installation docs, so we don't have repeated documentation around. Hopefully we will maintain that properly and up to date and when the documentation site is up an ready it will be easy to navigate through sections. What do you think?